charts/argocd-notifications/templates/secret.yaml

```
@@ -12,7 +12,7 @@ stringData:
     slack:
       token: {{ .Values.secret.notifiers.slack.token }}
       username: {{ .Values.secret.notifiers.slack.username }}
-      icon: {{ .Values.secret.notifiers.slack.icon }}
+      icon: "{{ .Values.secret.notifiers.slack.icon }}"
```

Believe the general way this is done in Helm is:
```suggestion
      icon: {{ .Values.secret.notifiers.slack.icon | quote }}
```